src/dxgi/dxgi_swapchain.cpp

```diff
@@ -171,7 +171,13 @@ namespace dxvk {
     pStats->PresentCount = m_presentCount;
     pStats->PresentRefreshCount = 0;
     pStats->SyncRefreshCount = 0;
+#ifdef _WIN32
     QueryPerformanceCounter(&pStats->SyncQPCTime);
+#else
+    auto now = dxvk::high_resolution_clock::now().time_since_epoch();
+    pStats->SyncQPCTime.QuadPart =
+      std::chrono::duration_cast<std::chrono::nanoseconds>(now).count();
```

Comment:
Any reason not to unconditionally use this path? On win32 we use QPC anyway, and this *should* return the unmodified value.

Reply:
That would not be correct, because `::now()` is divided by the frequency, and Windows expects this value not to be:
```cpp
    static inline time_point now() noexcept {
      // Keep the frequency static, this doesn't change at all.
      static const int64_t freq = getFrequency();
      const int64_t counter     = getCounter();

      const int64_t whole = (counter / freq) * period::den;
      const int64_t part  = (counter % freq) * period::den / freq;

      return time_point(duration(whole + part));
    }
```

Reply:
Meh, I forgot that QPC is some arbitrary bullshit.

I'd still rather have this abstracted away in util_time.h. Can do the same trick as with threads and subclass to add some new methods?

Reply:
I'll remove this for now to not block as per discussion

Reply:
Done.